### ISSUE TITLE
Create a routine triage strategy

### ISSUE THREAD

**Issue by seanmonstar** [hyperium/hyper#3214]
This is an action item of the [COE on hyper's surprise CVE](https://seanmonstar.com/post/715784167270596608/coe-surpise-hyper-cve). We have a pile of issues and pull requests that don't get regularly reviewed, and new issues that take a while to get some eyes. It would be beneficial to setup some strategy to do. We can make a more welcoming experience, promptly fix problems, identify more serious issues, and mentor new contributors.

The resolution of this issue should be a pull request adding to the `docs/` folder in this repo, outlining what the triage strategy is.

**Note**: anyone is encouraged to sign up a [triager](https://github.com/hyperium/hyper/blob/master/docs/MAINTAINERS.md#triagers), to help us out.

## Some questions to consider:

- How often should "routine" be?
- Should it be done synchronously?
- How do we keep ourselves accountable to doing it?
- How do we keep track of what has been triaged, and what next actions are for any issue or pull request?

## Some possible options include:

- Synchronous text meeting, such as in a Discord channel.
- Synchronous video meeting.
- Partial video, such as streaming a screen (like on Twitch or Discord), and others in text chat.
- Async, having a few people do it at a convenient time for them during the week, and send a report somewhere (Discord?)

For keeping track, we could create a `T-` sets of labels, like `T-needs-mvce`, `T-waiting-on-author`, etc. We can come up with a specified amount of time for something to be sitting in a certain "triage" state, before moving to another, or being closed.

## Triage process

These are just proposed things to do, so far:

- Make sure new issues have all the needed info from the reporter. This would benefit from a checklist (separate issue).
- Check if any issues waiting on author have received comments from them.
- Consider tagging a collaborator (or a volunteer to mentor) that can fix it, or review if a PR.

**Comment by seanmonstar** [comment]
cc @hyperium/hyper @hyperium/triage
